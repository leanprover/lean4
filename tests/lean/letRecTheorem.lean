def foo : Fin 5 :=
  let rec bla : 0 < 5 := by decide
  ⟨0, bla⟩
#print foo.bla
