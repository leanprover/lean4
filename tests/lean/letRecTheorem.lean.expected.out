theorem foo.bla : 0 < 5 :=
of_decide_eq_true (Eq.refl true)
